devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy

```diff
@@ -225,6 +234,19 @@ def refresh() {
 	delayBetween(commands,100)
 }
 
+def healthPoll() {
+	def cmds = refresh()
+	cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
```

Comment:
This is bad form – when sending multiple commands they must all be sent in the same object, a HubMultiAction, or the delays don't actually do anything.

The correct form would be sendHubCommand(response(refresh())) – response() is a helper that converts commands to HubAction objects. Unfortunately it looks like sendHubCommand only accepts single actions, so that has to be fixed first.